v19.2/performance.md

```diff
@@ -0,0 +1,101 @@
+---
+title: CockroachDB Performance
+summary: TBD
+toc: true
+toc_not_nested: true
+---
+
+CockroachDB delivers predictable throughput and latency at all scales on commodity hardware. This page provides an overview of the performance profiles you can expect, based on Cockroach Lab's extensive testing using industry-standard benchmarks like TPC-C and Sysbench.
+
+Instructions to independently reproduce the results listed here are coming soon. In the meantime, you can [simulate the benchmarking process in a local cluster](performance-benchmarking-with-tpc-c-local.html) or follow these [older instructions for benchmarking with TPC-C](performance-benchmarking-with-tpc-c.html).
+
+{{site.data.alerts.callout_success}}
+This document is about CockroachDB’s performance on benchmarks. For guidance on tuning real workloads, see [SQL Best Practices](performance-best-practices-overview.html), and for guidance on data location techniques to minimize network latency, see [Topology Patterns](topology-patterns.html).  
+{{site.data.alerts.end}}
+
+<!-- Let's hold off on this statement until we have reproducible steps.
+Once those are available, if you fail to achieve similar performance profiles, there is likely a problem in either the hardware, workload, or test design. We stand by these profile characteristics and provide reproduction steps for all published benchmarks. -->
+
+## Benchmarks
+
+### TPC-C
```

Comment:
I might move the benchmark descriptions below the fold and put the results up top. I think it will make the page read better.

Reply:
@jseldess until we have separate pages for explaining Sysbench and TPC-C, how about we move them to the bottom just above the performance limitations section. We can then link down within the page with a sentence like this benchmarks are explained here.

Reply:
Yeah, I think that'd work. Let's maybe rename the section `Benchmarks used`?

Reply:
done